mappings/net/minecraft/client/InteractionManager.mapping

```diff
@@ -1,17 +1,60 @@
 CLASS net/minecraft/class_504 net/minecraft/client/InteractionManager
 	FIELD field_2104 minecraft Lnet/minecraft/client/Minecraft;
+	FIELD field_2105 noTick Z
 	METHOD <init> (Lnet/minecraft/client/Minecraft;)V
 		ARG 1 minecraft
+	METHOD method_1705 resetBlockBreaking ()V
+	METHOD method_1706 update (F)V
+	METHOD method_1707 interactBlock (IIII)V
+		ARG 1 x
+		ARG 2 y
+		ARG 3 z
+		ARG 4 direction
 	METHOD method_1708 clickSlot (IIIZLnet/minecraft/class_54;)Lnet/minecraft/class_31;
 		ARG 1 windowId
 		ARG 2 slotId
 		ARG 3 button
 		ARG 4 shift
 		ARG 5 player
+	METHOD method_1709 onContainerRemove (ILnet/minecraft/class_54;)V
+		ARG 1 containerId
+		ARG 2 player
+	METHOD method_1710 setWorld (Lnet/minecraft/class_18;)V
+		ARG 1 world
+	METHOD method_1711 preparePlayer (Lnet/minecraft/class_54;)V
+		ARG 1 player
+	METHOD method_1712 interactItem (Lnet/minecraft/class_54;Lnet/minecraft/class_18;Lnet/minecraft/class_31;)Z
+		ARG 1 player
+		ARG 2 world
+		ARG 3 item
+	METHOD method_1713 interactBlockWithItem (Lnet/minecraft/class_54;Lnet/minecraft/class_18;Lnet/minecraft/class_31;IIII)Z
+		ARG 1 player
+		ARG 2 world
+		ARG 3 item
+		ARG 4 x
+		ARG 5 y
+		ARG 6 z
+		ARG 7 side
 	METHOD method_1714 interactEntity (Lnet/minecraft/class_54;Lnet/minecraft/class_57;)V
 		ARG 1 player
 		ARG 2 entity
+	METHOD method_1715 getReachDistance ()F
+	METHOD method_1716 breakBlock (IIII)Z
+		ARG 1 x
+		ARG 2 y
+		ARG 3 z
+		ARG 4 direction
+	METHOD method_1717 createPlayer (Lnet/minecraft/class_18;)Lnet/minecraft/class_54;
+		ARG 1 world
+	METHOD method_1718 preparePlayerRespawn (Lnet/minecraft/class_54;)V
+		ARG 1 player
 	METHOD method_1719 attackEntity (Lnet/minecraft/class_54;Lnet/minecraft/class_57;)V
 		ARG 1 player
 		ARG 2 target
 	METHOD method_1720 tick ()V
+	METHOD method_1721 onBlockBreakingAction (IIII)V
```

Comment:
[processBlockBreakingAction](https://maven.fabricmc.net/docs/yarn-1.20-pre4+build.2/net/minecraft/server/network/ServerPlayerInteractionManager.html#processBlockBreakingAction(net.minecraft.util.math.BlockPos,net.minecraft.network.packet.c2s.play.PlayerActionC2SPacket.Action,net.minecraft.util.math.Direction,int,int)) in Yarn